packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -330,6 +331,7 @@ export default class UsersRepo extends AbstractRepo<DbUser> {
             v."userId" = $1
             AND u._id != $1
             AND v."votedAt" > NOW() - INTERVAL '1.5 years'
+            AND v."cancelled" IS FALSE
```

Comment:
I would say `IS NOT TRUE`, which is safer (since `cancelled` can currently be `null`, in principle, and you don't want to drop those, since those aren't cancelled).